module-2/backend/week_one.md

```diff
@@ -5,30 +5,79 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+GET - retrieves information from the client
+POST - takes information from the app and puts it into the view
+PUT - similar to post but used when working from an existing file path
+DELETE - deletes 
+I don't know the fifth one.
+
 2. What is Sinatra?
+A minimalist web application framework that uses MVC set-up. Commonly uses ActiveRecord. Has similarities to Rails but doesn't have as much built in functionality.
+
 4. What is MVC?
+It stands for Model, View, Controller. The Model is the area that handles functionality, modeling the behavior that the application should perform, including the database interface and structure. The view is the place where all things that the client-side will see is held, commonly erb files that allow interaction through forms. Controller handles all HTTP requests and routes them to the appropriate location and calls the correct Model methods on these interactions. 
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+The conventions allow better collaboration between developers. They also have the benefit of making complex design decisions for us in advance, so we don't have to re-invent the wheel to solve common problems in application development each time that we build a new application. They save time and energy.  
+
 6. What types of variables are accessible in our view templates without explicitly passing them?
+params
+
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
+  @count = 1 
   
   ```ruby
-  get '/horses' do
+  get '/horses' do  
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+name = 'Mr. Ed'

```

Comment:
Close, but this wouldn't allow us to access it in the view. We need to pass our local variables explicitly.

```ruby
get '/horses' do
  name = "Mr. Ed"
  erb :index, locals: {name: name}
end
```

Alternatively, without assigning Mr. Ed to a local variable:

```ruby
get '/horses' do
  erb :index, locals: {name: "Mr. Ed"}
end
```